source/js/buyers-guide/components/creep-vote/creep-vote.scss

```diff
@@ -70,14 +70,4 @@ $btn-shadow-width: 3px;
   .help-text {
     font-style: italic;
   }
-
```

Comment:
It looks like this file is still part of your PR - please reset this file to what it is on the master branch.